File: ezSCORES/UI/ezscores_mobile/lib/models/DTOs/matchesByDateDto.dart
import 'package:ezscores_mobile/models/DTOs/matchDto.dart';
import 'package:ezscores_mobile/models/competitions.dart';
import 'package:json_annotation/json_annotation.dart';

part 'matchesByDateDto.g.dart';

@JsonSerializable()
class MatchesByDateDTO {
	Competitions? competition;
	List<MatchDTO>? matches;

	MatchesByDateDTO(this.competition,this.matches,);

	factory MatchesByDateDTO.fromJson(Map<String, dynamic> json) => _$MatchesByDateDTOFromJson(json);

	Map<String, dynamic> toJson() => _$MatchesByDateDTOToJson(this);
}

File: ezSCORES/ezSCORES.Model/DTOs/MatchesByDateDTO.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model.DTOs
{
	public  class MatchesByDateDTO
	{
		public Competitions Competition { get; set; }
		public List<MatchDTO> Matches { get; set; }
	}
}
